libbpf-rs/src/program.rs

```diff
@@ -621,6 +624,8 @@ pub struct Output<'dat> {
     pub context: Option<&'dat mut [u8]>,
     /// Output data filled by the program.
     pub data: Option<&'dat mut [u8]>,
+    /// average per repetition in ns
+    pub duration: u32,
```

Comment:
Could we use a `std::time::Duration` here instead?

Reply:
probably can.